packages/react-core/src/demos/Filters/examples/FilterAttributeSearch.tsx

```diff
@@ -0,0 +1,701 @@
+    <Menu
+      ref={attributeMenuRef}
+      onSelect={(_ev, itemId) => {
+        setActiveAttributeMenu(itemId?.toString() as 'Servers' | 'Status' | 'Location');
+        setIsAttributeMenuOpen(!isAttributeMenuOpen);
+      }}
+    >
+      <MenuContent>
+        <MenuList>
+          <MenuItem itemId="Servers">Servers</MenuItem>
+          <MenuItem itemId="Status">Status</MenuItem>
+          <MenuItem itemId="Location">Location</MenuItem>
+        </MenuList>
+      </MenuContent>
+    </Menu>
+  );
+
+  const attributeDropdown = (
+    <div ref={attributeContainerRef}>
+      <Popper
+        trigger={attributeToggle}
+        popper={attributeMenu}
+        appendTo={attributeContainerRef.current || undefined}
+        isVisible={isAttributeMenuOpen}
+      />
+    </div>
+  );
+
+  // Set up pagination and toolbar
+  const toolbarPagination = (
+    <Pagination
+      titles={{ paginationTitle: 'Attribute search pagination' }}
+      perPageComponent="button"
+      itemCount={repositories.length}
+      perPage={10}
+      page={1}
+      widgetId="attribute-search-mock-pagination"
+      isCompact
+    />
+  );
+
+  const toolbar = (
+    <Toolbar
+      id="attribute-search-filter-toolbar"
+      clearAllFilters={() => {
+        setSearchValue('');
+        setStatusSelection('');
+        setLocationSelections([]);
+      }}
+    >
+      <ToolbarContent>
+        <ToolbarItem>{toolbarBulkSelect}</ToolbarItem>
+        <ToolbarToggleGroup toggleIcon={<FilterIcon />} breakpoint="xl">
+          <ToolbarGroup variant="filter-group">
+            <ToolbarItem>{attributeDropdown}</ToolbarItem>
+            <ToolbarFilter
+              chips={searchValue !== '' ? [searchValue] : ([] as string[])}
+              deleteChip={() => setSearchValue('')}
+              deleteChipGroup={() => setSearchValue('')}
+              categoryName="Name"
+              showToolbarItem={activeAttributeMenu === 'Servers'}
+            >
+              {searchInput}
+            </ToolbarFilter>
+            <ToolbarFilter
+              chips={statusSelection !== '' ? [statusSelection] : ([] as string[])}
+              deleteChip={() => setStatusSelection('')}
+              deleteChipGroup={() => setStatusSelection('')}
+              categoryName="Status"
+              showToolbarItem={activeAttributeMenu === 'Status'}
+            >
+              {statusSelect}
+            </ToolbarFilter>
+            <ToolbarFilter
+              chips={locationSelections}
+              deleteChip={(category, chip) => onLocationMenuSelect(undefined, chip as string)}
+              deleteChipGroup={() => setLocationSelections([])}
+              categoryName="Location"
+              showToolbarItem={activeAttributeMenu === 'Location'}
+            >
+              {locationSelect}
+            </ToolbarFilter>
+          </ToolbarGroup>
+        </ToolbarToggleGroup>
+        <ToolbarItem variant="pagination">{toolbarPagination}</ToolbarItem>
```

Comment:
Do these need pagination in the toolbar if it doesn't do anything?

Reply:
Design confirmed wanting the paginations in on the previous filter demos PR